src/EA.Weee.DataAccess.Tests.Integration/NotesDataAccessIntegration.cs

```diff
@@ -571,7 +574,123 @@ public async Task GetNoteCountByStatusAndAatf_GivenStatusAndAatf_ShouldReturnCor
             }
         }
 
-        private async Task<Note> SetupSingleNote(WeeeContext context, DatabaseWrapper database, NoteType noteType = null)
+        [Fact]
+        public async Task GetNotesToTransfer_GivenSchemeAndCategories_NotesShouldBeReturned()
+        {
+            using (var database = new DatabaseWrapper())
+            {
+                var context = database.WeeeContext;
+                var dataAccess = new EvidenceDataAccess(database.WeeeContext, A.Fake<IUserContext>());
+
+                var organisation1 = ObligatedWeeeIntegrationCommon.CreateOrganisation();
+                var scheme = ObligatedWeeeIntegrationCommon.CreateScheme(organisation1);
+
+                // to be found matching category, scheme and status
+                var note1ToBeFound = await SetupSingleNote(context, database, NoteType.EvidenceNote, scheme);
+                note1ToBeFound.UpdateStatus(NoteStatus.Submitted, context.GetCurrentUser());
+                note1ToBeFound.UpdateStatus(NoteStatus.Approved, context.GetCurrentUser());
+                note1ToBeFound.NoteTonnage.Add(new NoteTonnage(WeeeCategory.ConsumerEquipment, 1, null));
+                note1ToBeFound.NoteTonnage.Add(new NoteTonnage(WeeeCategory.GasDischargeLampsAndLedLightSources, 2, null));
+
+                context.Notes.Add(note1ToBeFound);
+
+                // note not to be found null category, matching scheme and status
+                var note2ToNotBeFound = await SetupSingleNote(context, database, NoteType.EvidenceNote, scheme);
+                note2ToNotBeFound.UpdateStatus(NoteStatus.Submitted, context.GetCurrentUser());
+                note2ToNotBeFound.UpdateStatus(NoteStatus.Approved, context.GetCurrentUser());
+                note2ToNotBeFound.NoteTonnage.Add(new NoteTonnage(WeeeCategory.ConsumerEquipment, null, null));
```

Comment:
it's matching the category accepted but it's not matching the tonnage required so maybe a bit more specific on the comment, not important at all

Reply:
Updated description.